client/dashboard/src/pages/toolsets/Toolset.tsx

```diff
@@ -246,7 +247,8 @@ export function ToolsetView({
 
   let toolsToDisplay: ToolDefinition[] = grouped
     .filter((group) => selectedGroups.includes(group.key))
-    .flatMap((group) => group.tools);
+    .flatMap((group) => group.tools)
+    .filter((tool) => tool.type !== PromptTemplateKind.Prompt);
```

Comment:
Hm, this looks odd? I thought tool.type was supposed to be either "http" or "higher order tool" (or equivalent)? looking at the ToolDefinition type i see type can be either of the PromptTemplateKinds , but that doesn't really make sense. A tool actually cant be a prompt. So maybe that would help clean all this up by adjusting that type def